src/js/components/Button/stories/Disabled.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Box, Button, Grommet } from 'grommet';
+
+export const Disabled = () => (
+  <Grommet
+    theme={{ global: { control: { disabled: { cursor: 'not-allowed' } } } }}
```

Comment:
Instead of adding a new story, can this theme just be added to this story https://storybook.grommet.io/?path=/story/controls-button-custom-themed-ts-custom--ts-custom?